src/common/assert.ts

```diff
@@ -2,28 +2,50 @@
 // Licensed under the MIT license.
 
 import { CommonError } from ".";
-import { Err, Ok, Result, ResultUtils } from "./result";
+import { Result, Ok, Err, ResultUtils } from "./result";
```

Comment:
You don't get warnings about these having to be sorted alphabetically?

Reply:
Talked offline, Looks like my machine didn't have TSLint installed. I should look into adding it to `extensions.json`.